runtime/j9vm/javanextvmi.cpp

```diff
@@ -763,15 +763,62 @@ JVM_IsImplicitlyConstructibleClass(JNIEnv *env, jclass cls)
 JNIEXPORT jboolean JNICALL
 JVM_IsNullRestrictedArray(JNIEnv *env, jobject obj)
```

Comment:
it doesn't need to be done in this PR, but all Valhalla related functions should be moved to a different file (perhaps valhallavmi.cpp). For large features like valhalla, loom, etc. we should try to avoid bundling  in the same file like we did in the past.

Reply:
I'll do this in a separate pr.